src/routes/about/+page.js

```diff
@@ -0,0 +1,352 @@
+        ]
+      }
+    ]
+  },
+  {
+    id: 7941918,
+    name: "Jitse Niesen",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "External plugins maintainer"
+        ]
+      }
+    ]
+  },
+  {
+    id: 42411448,
+    name: "Juan Sebastian Bautista",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Improvements to the Windows installer, QtConsole and Spyder's UI/UX"
+        ]
+      }
+    ]
+  },
+  {
+    id: 5204788,
+    name: "Hendrik D. Louzada",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Backend for the Remote client plugin"
+        ]
+      }
+    ]
+  },
+  {
+    id: 5027583,
+    name: "Andrés Montoya",
+    role: "Grpahic and web designer",
+    tooltip: [
+      {
+        list: [
+          "Developer and maintainer of this website",
+          "UI/UX improvements",
+          "Social media graphics designer"
+        ]
+      }
+    ]
+  },
+];
+
+const pastContributors = [
+  {
+    id: 1311787,
+    name: "Pierre Raybaut",
+    role: "Spyder Creator",
+    tooltip: [
+      {
+        list: [
+          "Started Spyder in 2009",
+          "Lead maintainer until 2013",
+        ]
+      }
+    ]
+  },
+  {
+    id: 1878982,
+    name: "Edgar Margffoy",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "LSP support for the entire application",
+          "Re-architecture plugin registration and interactions",
+          "New architecture for the Run plugin",
+          "Add global registries for actions, menus and toolbars",
+        ]
+      }
+    ]
+  },
+  {
+    id: 50221806,
+    name: "Isabela Presedo-Floyd",
+    role: "Designer",
+    tooltip: [
+      {
+        list: [
+          "New color system and palette for the application",
+          "Logo redesign",
+          "Many UI/UX improvements"
+        ]
+      }
+    ]
+  },
+  {
+    id: 3627835,
+    name: "Gonzalo Peña-Castellanos",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Redesign of the plugins API",
+          "Creator of Qtpy",
+          "Test and improve configuration system",
+          "Many improvements to the Editor"
+        ]
+      }
+    ]
+  },
+  {
+    id: 18587879,
+    name: "Juanita Gómez",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Docs maintainer",
+          "Main content creator for Spyder's YouTube channel",
+          "Community manager"
+        ]
+      }
+    ]
+  },
+  {
+    id: 10170372,
+    name: "Jean-Sébastien Gosselin",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Creator of the Plots plane",
+          "Many improvements across the entire application",
+        ]
+      }
+    ]
+  },
+  {
+    id: 20992645,
+    name: "Stephannie Jimenez",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Spyder-terminal maintainer",
```

Comment:
```suggestion
          "Spyder-Terminal maintainer",
```

Consistent/correct Proper Case capitalization